bsip-0071.md

```diff
@@ -51,59 +75,126 @@ In my view, a good way to handle bad debt need to follow below principles:
+Some may perceive it as confusing or unfair that borrowing and force settlement may refer to different prices. 
+However, in a period where the least collateralized short position has insufficient collateral in terms of FP_m, 
+it would be dangerous to use the resulting settlement_price for borrowing as well, because that would allow creation 
+of short positions that are undercollateralized from the start.
 
 ## Least collateralized short
 
 Two variants for the *call execution price* have been discussed.
 
-Variant A (as specified here) prefers to match the bad debt positions first, hoping that this way the bad debt will be resolved quickly. This also means that if P<sub>gs</sub> moves away from the market price, **no** margin calls will be filled, not even those with a collateral ratio that would allow them being matched with market orders. This can lead to more bad debt piling up instead of resolving at least some debt positions through margin calls.
+Variant A (as specified here) prefers to match the bad debt positions first, hoping that this way the bad debt will 
+be resolved quickly. This also means that if P<sub>gs</sub> moves away from the market price, **no** margin calls will 
+be filled, not even those with a collateral ratio that would allow them being matched with market orders. 
+This can lead to more bad debt piling up instead of resolving at least some debt positions through margin calls.
+
+Variant B would have let bad debt positions matched at their own collateral ratio while matching other margin calls 
+as before at FP<sub>M</sub> &times; MSSR, hoping to improve the overall collateralization. This also means that debt 
+positions with better collateral ratio would have been preferred over bad debt, meaning that bad debt positions will 
+remain sitting on the market for a longer time.
+
+## The threshhold price
+
+With this BSIP the threshhold price is defined by the asset owner. This may not be considered fair. A suitable 
+future extension is to let asset **holders** have a say in the threshhold price discovery. This could be done by
+introducing a voting procedures where holders of the SmartCoin can participate, with their voting weight reflecting their
+holdings of the SmartCoin in question. This procedure needs precise definition in a new BSIP.
+
+## Voting power
+
+As of right now, a SmartCoin that has BTS as its collateral provides the full amount of collateral as voting power. 
+This is not considered to be fair since it is borrowed voting power and gives incentives to squeeze out as much as 
+possible from a margin position to increase voting weight (borrow, sell bitasset on market, borrow more, and so on). 
+This is adressed by adjusting voting power of a position, and the suggested solution ensures that the voting power of 
+a margin position holders remains unchanged, with the assumption that he sells the newly borrowed bitasset on the 
```

Comment:
Wouldn't the voting power of margin position holders decrease after implementation since the 1x CR vote weight would be null, only > 1x CR would count as vote weight.

Reply:
refined the wording